packages/web-components/src/components/masthead/left-nav-menu.ts

```diff
@@ -115,8 +131,14 @@ class DDSLeftNavMenu extends FocusMixin(LitElement) {
         </div>
       </button>
       <ul class="${prefix}--side-nav__menu" role="menu">
-        <li class="bx--side-nav__menu-item bx--masthead__side-nav--submemu-back" role="none">
-          <a class="bx--side-nav__link" role="menuitem" @click="${handleClickExpando}">
+        <li class="bx--side-nav__menu-item bx--masthead__side-nav--submemu-back">
+          <a
+            class="bx--side-nav__link"
+            role="menuitem"
+            @keydown="${handleKeydownExpando}"
+            @click="${handleClickExpando}"
+            tabindex="0"
```

Comment:
Weighing in; `<a>` works as `tabindex="0"` by default.

Reply:
It doesn't if you remove the `href`, @asudoh 

https://codesandbox.io/s/wizardly-matsumoto-4r1t5?file=/src/App.js

Reply:
In such case, do you want to add `href` back? The `<a>` features we lose are more than focusing if we remove `href`.

Reply:
This element never had an `href`  attribute. How can we use it?

Reply:
Would you want to try something like [this](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.16.1/packages/web-components/src/components/masthead/left-nav.ts#L136)?

Reply:
I did it and it makes it stop working. Do you know why we're using an `anchor` tag instead of a `button type="button"`?

Reply:
I don't know. If `<button>` is OK for us I think it'll definitely be a good idea.

Reply:
@asudoh coming back on this again before any changes: the React version is currently using an `a` tag. Also, changing the tag would require updating the styles as well. So I'm wondering if we need to change it.

Reply:
In such case, do you want to deep-dive why adding `href` broke things?

Reply:
@asudoh I've been testing it but couldn't make it work with the `href`.